Sources/Sentry/SentryOptions.m

```diff
@@ -314,6 +316,12 @@ - (BOOL)validateOptions:(NSDictionary<NSString *, id> *)options
     }
 }
 
+- (SentrySdkInfo *)sdkInfo
+{
+    return [[SentrySdkInfo alloc] initWithName:SentryMeta.sdkName
```

Comment:
`m`: Can we please add a test that verifies that the options don't store this info anymore? Basically, a test that checks when changing the value of SentryMeta that the returned value here is impacted by this.

Reply:
Never mind, previously it was only set once, now it's a getter always getting the current version of SentryMeta so yea good test to have 👍